core/modules/filter/filter.module

```diff
@@ -316,6 +324,40 @@ function filter_format_save($format) {
     $format->filters[$name] = $filter;
   }
 
+  return $format;
+}
+
+/**
+ * Saves a text format object to the database.
```

Comment:
Aren't these config? Not in the database?

Reply:
Yeah would be better to say config.

Reply:
updated.